CONTRIBUTING.md

```diff
@@ -1,3 +1,50 @@
+## Set Up a Development Environment
+
+1. `$ git clone git@github.com:ruby/debug.git`
+2. `$ bundle install`
+3. `$ rake` - this will
+    - Compile the C extension locally (which can also be done solely with `rake compile`).
+    - Run tests.
+    - Re-generate `README.md`.
+
+If you spot any problem, please open an issue.
+
+## Run Tests
+
+### Run all tests
+
+```bash
+$ rake test
+# or
+$ ruby bin/test-unit.rb
+```
+
+### Run specific test(s)
+
+
+```bash
+$ ruby bin/test-unit.rb -t '/TestClass/'
```

Comment:
@ono-max can you add your suggestion here? then I can just commit it 😄

Reply:
```suggestion
$ ruby bin/test-unit.rb -t '/TestClass/'
# or
$ ruby bin/test-unit.rb debug/break_test.rb
```